GameStore/Views/Home/Index.cshtml

```diff
@@ -97,10 +97,17 @@
         })
         strResult += "</select><div id=ganreGamesTable></div></div>";
         $("#tableBlock").html(strResult);
+        
+        if (genres != null && genres.length > 0) {
```

Comment:
If I am not making a mistake, you can perform such check just as boolean in C#.
`if(genres && genres.length > 0) {`

Reply:
Optimized